atlasdb-workload-server/src/main/java/com/palantir/atlasdb/workload/store/AtlasDbInteractiveTransaction.java

```diff
@@ -91,6 +100,36 @@ public void delete(String table, WorkloadCell workloadCell) {
                 workloadCell);
     }
 
+    @Override
+    public List<ColumnValue> getRowColumnRange(String table, Integer row, ColumnRangeSelection columnRangeSelection) {
+        return run(
+                tableReference -> {
+                    // Having a non-configurable batch hint is a bit iffy, but suffices as this won't be used in
+                    // production.
+                    Map<byte[], Iterator<Entry<Cell, byte[]>>> iterators = transaction.getRowsColumnRangeIterator(
```

Comment:
We currently ignore the case where this returns multiple rows, but given that we only request a single row, that would be a bug. We could add a precondition here to surface that bug, if here's the relevant place to do so

Reply:
Yep, added a precondition for returning just one.